### PR TITLE
Feature/profile layout

### PR DESCRIPTION
**Pull request by KyleWong2510**
## Pull request checklist
Please check if your PR fulfills the following requirements:
- [ ] Lint (`npm run lint`) has passed locally and any fixes were made for failures

## Pull request type
 
 
Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Testing
- [ ] Other (please describe): 

## What is the current behavior?

- No UserInfo component
- No DeliveryInfo component

## What is the new behavior?

- Added both components so our data has somewhere to land
- Added conditional render functionality
- UserInfo populates with data fetched from login

## Does this introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by edignot** [comment]
Great progress

File: code/web/src/modules/user/UserInfo.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    super() 
+    this.state = {
+      isEditing: false,
+      emailInput: '',
+      userDescription: '',
+      shippingAddressInput: ''
+    }
+  }
+
+  componentDidMount = () => {
```
Comment (KyleWong2510) [inline code/web/src/modules/user/UserInfo.js:21]: Sets local state on mounting of component (when the component renders to the page) using fetched data from the login network request.

File: code/web/src/modules/user/UserInfo.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+      shippingAddressInput: user.shippingAddress || 'Add your shipping address'
+    })
+  }
+
+  toggleEdit = () => {
+    this.setState({ isEditing: !this.state.isEditing })
+  }
+
+  handleChange = (e) => {
+    const { name, value } = e.target
```
Comment (KyleWong2510) [inline code/web/src/modules/user/UserInfo.js:35]: Sets local state to reflect changes made in the inputs

File: code/web/src/modules/user/UserInfo.js [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+    this.setState({ [name]: value })
+  }
+
+  renderInputs = () => {
```
Comment (KyleWong2510) [inline code/web/src/modules/user/UserInfo.js:39]: Renders inputs while editing

File: code/web/src/modules/user/UserInfo.js [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+          type='text'
+          name='shippingAddressInput'
+          value={this.state.shippingAddressInput}
+          onChange={e => this.handleChange(e)}
+        />
+      </section>
+    )
+  }
+
+  renderInfo = () => {
```
Comment (KyleWong2510) [inline code/web/src/modules/user/UserInfo.js:65]: Renders info when not editing

File: code/web/src/modules/user/UserInfo.js [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+        <section>
+          <H5>Shipping Address</H5>
+          <p>{this.state.shippingAddressInput}</p>
+        </section>
+      </section>
+    )
+  }
+
+  render() {
+    return (
```
Comment (KyleWong2510) [inline code/web/src/modules/user/UserInfo.js:81]: Conditionally renders buttons and inputs based on whether or not the user is editing

File: code/web/src/modules/user/UserInfo.js (hunk 1 above)
Comment (sagemlee) [reply code/web/src/modules/user/UserInfo.js:21]: Thank You for inputting all of these notes, it is helpful when looking at the Delivery info file as well.

File: code/web/src/modules/user/UserInfo.js (hunk 2 above)
Comment (sagemlee) [reply code/web/src/modules/user/UserInfo.js:35]: I like that this one is easy to follow considering the method is called "setState"

File: code/web/src/modules/user/UserInfo.js (hunk 5 above)
Comment (sagemlee) [reply code/web/src/modules/user/UserInfo.js:81]: I would be interested in walkthrough this with you another time, It looks similar to an if than statement, but I isnt, so Im a little confused.

**Pull request merged**